src/nvme/ioctl.h

```diff
@@ -2714,6 +2714,7 @@ int nvme_resv_report(int fd, __u32 nsid, bool eds, __u32 len,
  * @slba:	Starting logical block address
  * @zsa:	Zone send action
  * @select_all:	Select all flag
+ * @zsaso:  Zone Send Action Specific Option
```

Comment:
The description is not aligned.

Reply:
Thanks. few of docs they using spaces, so I thought this use same.

Reply:
It seems most of the doc is somehow aligned. That's why I think it makes sense to have here as well.